### ISSUE TITLE
AWS upload breaks for large files

### ISSUE THREAD

**Issue by rtisma** [overture-stack/score#120]
DCCBIO-31 ticket

**Comment by rtisma** [comment]
Was able to replicate the issue. The issue was resolved by adding the following lines to my score-client/conf/application.properties

```
logging.level.bio.overture.score=DEBUG
logging.level.org.springframework.retry=TRACE
logging.level.org.springframework.web=DEBUG
logging.level.com.amazonaws.services=TRACE
transport.memory=5
client.connectTimeoutSeconds=600
client.readTimeoutSeconds=600 
```

The issue was that after uploading all the parts, the final step, `finalizeUpload` which essentially does a concatenation of all the parts into one giant part, was timing out. It was most likely timing out because there were so many parts. The actual operation needs about 2 minutes, so as a safety net, i suggest using 10 minutes (600 seconds) for the time out. 

The logging configs are not necessary for operation, however they are nice to have when things go wrong.

**Comment by rosibaj** [comment]
Discussed in Sprint Planning:
- DCC-BIO-31 ticket in JIRA
- We do want to do this ticket, but to unblock bioinformatics we can upload through AWS
- objectid is in song, upload is failing.  
- @rtisma Ask bioinformatics for md5 sum and generate the .meta file 
- @junjun-zhang, once Rob provides the .meta files, your team can use AWS-cli to upload these files until this can be prioritized
